pallets/automation-time/src/tests.rs

```diff
@@ -330,11 +362,15 @@ fn schedule_xcmp_through_proxy_works() {
 			RuntimeOrigin::signed(proxy_account.clone()),
 			provided_id,
 			ScheduleParam::Fixed { execution_times: vec![SCHEDULED_TIME] },
-			PARA_ID.try_into().unwrap(),
-			NATIVE,
-			MultiLocation::new(1, X1(Parachain(PARA_ID.into()))).into(),
+			Box::new(destination.clone().into()),
+			Box::new(MultiLocation::default().into()),
+			Box::new(AssetPayment {
+				asset_location: destination.into(),
+				amount: 10,
+			}),
 			call.clone(),
 			Weight::from_ref_time(100_000),
+			Weight::from_ref_time(200_000),
```

Comment:
Can you explain the reason of changing this value from 100_000 to 200_000?

Do we need two lines here?
```
Weight::from_ref_time(100_000),
Weight::from_ref_time(200_000),
```

Reply:
Because the schedule_xcmp_task_through_proxy function definition now includes the `overall_weight` parameter, I need to pass the parameter value here.

Reply:
okay, got it. That makes sense.